### PR TITLE
Call jitsi widgets 'group calls'

### PR DESCRIPTION
**Pull request by dbkr**
So they feel a bit more like first class citizens

### CODE REVIEW

File: src/TextForEvent.js [hunk 1]
```diff
@@ -475,6 +475,10 @@ function textForWidgetEvent(event) {
     const {name: prevName, type: prevType, url: prevUrl} = event.getPrevContent();
     const {name, type, url} = event.getContent() || {};
 
+    if (type === 'jitsi' || prevType === 'jitsi') {
```
Comment (turt2live) [inline src/TextForEvent.js:478]: Type can also be `m.jitsi`. We have a WidgetType class somewhere which does the matching for you

**Review by turt2live — COMMENTED** [review]
Otherwise lgtm

File: src/TextForEvent.js (hunk 1 above)
Comment (dbkr) [reply src/TextForEvent.js]: huh :/ Thanks.

File: src/TextForEvent.js [hunk 2]
```diff
@@ -475,6 +476,13 @@ function textForWidgetEvent(event) {
     const {name: prevName, type: prevType, url: prevUrl} = event.getPrevContent();
     const {name, type, url} = event.getContent() || {};
 
+    const prevWidgetType = WidgetType.fromString(prevType);
+    const widgetType = WidgetType.fromString(type);
+
+    if (widgetType === WidgetType.JITSI || prevWidgetType === WidgetType.JITSI) {
```
Comment (turt2live) [inline src/TextForEvent.js:482]: or alternatively: `WidgetType.JITSI.matches(type) || WidgetType.JITSI.matches(prevType)`

**Review by turt2live — APPROVED** [review]
lgtm either way (this widget type class is probably going to disappear eventually anyways)

**Pull request merged**